dowhy/gcm/util/graphviz.py

```diff
@@ -0,0 +1,72 @@
+import os
+import tempfile
+from typing import Optional, Dict, Tuple, Any
+
+import graphviz
+import networkx as nx
+import numpy as np
+from matplotlib import image, pyplot
+
+
+def plot_causal_graph_graphviz(causal_graph: nx.Graph,
```

Comment:
since pygraphviz sometimes is problematic to install, the current code in dowhy has a generic plot and falls back to matplotlib if the user does not have pygraphviz. (see https://github.com/py-why/dowhy/blob/0232fb433a0486dc71b0d9f53d75acc11516095c/dowhy/causal_graph.py#L94)

Is it possible to add that here too? The risk is that people who do not have graphviz will encounter an exception in the  notebooks.

Reply:
ah, I see that you do exactly this in the next file. Awesome!